src/components/products/Filter.jsx

```diff
@@ -0,0 +1,290 @@
+          <li className="filter-section-title">PRICE RANGE</li>
+          <div class="slidecontainer">
+            <input
+              type="range"
+              min="1000"
+              max="4000"
+              step="500"
+              value={FilterState.priceRange}
+              class="slider"
+              id="myRange"
+              onChange={(e) =>
+                FilterDispatch({
+                  type: "PRICE_RANGE",
+                  payload: Number(e.target.value),
+                })
+              }
+            />
+            <div class="span-text-slider">
+              <span>1k</span>
+              <span>2k</span>
+              <span>3k</span>
+              <span>4k</span>
+            </div>
+          </div>
+          <div className="filter-divider"></div>
+          <li className="filter-section-title">SORT</li>
+          <li className="list-item filter-item">
+            <label>
+              <input
+                className="filter-radio"
+                type="radio"
+                name="sort"
+                checked={FilterState.sortByPrice === "PRICE_LOW_TO_HIGH"}
+                value="PRICE_LOW_TO_HIGH"
+                onClick={() =>
+                  FilterDispatch({
+                    type: "SORT_BY_PRICE",
+                    payload: "PRICE_LOW_TO_HIGH",
+                  })
+                }
+              />
+              Price low to high
+            </label>
+            <label>
+              <input
+                className="filter-radio"
+                type="radio"
+                name="sort"
+                checked={FilterState.sortByPrice === "PRICE_HIGH_TO_LOW"}
+                value="PRICE_HIGH_TO_LOW"
+                onClick={() =>
+                  FilterDispatch({
+                    type: "SORT_BY_PRICE",
+                    payload: "PRICE_HIGH_TO_LOW",
+                  })
+                }
+              />
+              Price high to low
+            </label>
+          </li>
+
+          <div className="filter-divider"></div>
+
+          <li className="filter-section-title">IN STOCK</li>
+          <li className="list-item filter-item">
+            <label>
+              <input
+                className="filter-radio"
+                type="checkbox"
+                name="inStock"
+                checked={FilterState.includeOutOfStock}
+                value="INCLUDE_OUT_OF_STOCK"
+                onClick={() =>
+                  FilterDispatch({
+                    type: "INCLUDE_OUT_OF_STOCK",
+                    payload: false,
+                  })
+                }
+              />
+              Include out of stock
+            </label>
+          </li>
+
+          <div className="filter-divider"></div>
+          <li className="filter-section-title">CATEGORIES</li>
+          <li className="list-item filter-item">
+            <label>
+              <input
+                className="filter-checkbox"
+                type="radio"
```

Comment:
Categories should be of type checkbox instead of radio.
Let's say I want to buy Full sleeves t-shirt & Hoodies.

Attaching an image for reference
![x](https://user-images.githubusercontent.com/95222845/163530496-8d2d4638-ada2-43d6-80e4-3a4a8b4af0b2.png)